src/main.ts

```diff
@@ -165,6 +169,45 @@ const command: Command = {
 		else {
 			return compile(config(configArgs), options);
 		}
+	},
+	eject(helper: Helper) {
+		const ejectOutput: EjectOutput = {
+			npm: {
+				devDependencies: {
```

Comment:
good call @mwistrand. Might be worth us looking into loading the `package.json` in here and pulling out versions from named requirements perhaps. For beta I think this will be fine though.